packages/p2p/src/stores/general-store.js

```diff
@@ -374,6 +379,23 @@ export default class GeneralStore extends BaseStore {
         this.order_table_type = order_table_type;
     }
 
+    @action.bound
+    setP2PConfig() {
+        const { floating_rate_store } = this.root_store;
+        requestWS({ website_status: 1 }).then(response => {
+            if (!!response && response.error) {
+                floating_rate_store.setApiErrorMessage(response.error.message);
+            } else {
+                const { p2p_config } = response.website_status;
```

Comment:
```suggestion
                const { fixed_rate_adverts, float_rate_adverts, float_rate_offset_limit, fixed_rate_adverts_end_date } = response.website_status.p2p_config;
```
Nit: cleaner this way

Reply:
Thank you for the suggestion. Will make the change